sdcm/provision/scylla_yaml/cluster_builder.py

```diff
@@ -0,0 +1,198 @@
+            return None
+        return bool(self.params.get('experimental'))
+
+    @property
+    def enable_ipv6_dns_lookup(self) -> bool:
+        return self._is_ip_ssh_connections_ipv6
+
+    @property
+    def authenticator(self) -> Optional[str]:
+        if self._is_authenticator_valid:
+            return self._authenticator
+        return None
+
+    @property
+    def saslauthd_socket_path(self) -> Optional[str]:
+        if self._is_authenticator_valid:
+            return '/run/saslauthd/mux'
+        return None
+
+    @property
+    def authorizer(self) -> Optional[str]:
+        if self._authorizer in ['AllowAllAuthorizer', 'CassandraAuthorizer']:
+            return self._authorizer
+        return None
+
+    @property
+    def alternator_port(self) -> Optional[str]:
+        return self.params.get('alternator_port')
+
+    @property
+    def alternator_write_isolation(self) -> Optional[str]:
+        return "always_use_lwt" if self.params.get('alternator_port') else None
+
+    @property
+    def alternator_enforce_authorization(self) -> bool:
+        return bool(self.params.get('alternator_enforce_authorization'))
+
+    @property
+    def internode_compression(self) -> Optional[str]:
+        return self.params.get('internode_compression')
+
+    @property
+    def endpoint_snitch(self) -> Optional[str]:
+        """
+        Comes from get_endpoint_snitch
+        """
+        if snitch := self.params.get('endpoint_snitch'):
+            return snitch
+        if self._multi_region:
+            return self._default_endpoint_snitch
+        return None
+
+    @property
+    def ldap_attr_role(self) -> Optional[str]:
+        return 'cn' if self._is_ldap_authorization else None
+
+    @property
+    def ldap_bind_dn(self) -> Optional[str]:
+        if self._is_msldap_authorization:
+            return self._ms_ldap_bind_dn
+        if self._is_openldap_authorization:
+            return self._open_ldap_bind_dn
+        return None
+
+    @property
+    def ldap_bind_passwd(self) -> Optional[str]:
+        if self._is_msldap_authorization:
+            return self._ms_ldap_bind_passwd
+        if self._is_openldap_authorization:
+            return self._open_ldap_bind_passwd
+        return None
+
+    @property
+    def ldap_url_template(self) -> Optional[str]:
+        if self._is_msldap_authorization:
+            server_port = self._ms_ldap_server_address_port
+            ldap_filter = 'member=CN={USER}'
+        elif self._is_openldap_authorization:
+            server_port = self._open_ldap_server_address_port
+            ldap_filter = 'uniqueMember=uid={USER},ou=Person'
+        else:
+            return None
+        return f'ldap://{server_port}/{self._ldap_base_dn}?cn?sub?({ldap_filter},{self._ldap_base_dn})'
+
+    @property
+    def _is_msldap_authorization(self):
+        return self.params.get('use_ldap_authorization') and self.params.get('use_ms_ad_ldap')
```

Comment:
I think it must use "_is_ldap_authorization", because otherwise it duplicates it.

Reply:
Here is what I meant:

```
return self._is_ldap_authorization and self.params.get('use_ms_ad_ldap')
```